src/core_plugins/kibana/public/context/query_parameters/actions.js

```diff
@@ -43,7 +46,13 @@ export function QueryParameterActionsProvider() {
     )
   );
 
+  const addTermsFilter = (state) => (field, values, operation) => {
```

Comment:
I actually intended to convey that this adds a `terms` filter in the ES sense, but just noticed that it actually adds either an `exists` or a `phrase` filter. So the name is really misleading. I'll change it to `addFilter`.